doc/source/conf.py

```diff
@@ -91,6 +91,25 @@
   "tasklist",
 ]
 
+# Workaround to suppress warnings for missing cross-reference targets when using
+# internal page links with the MyST parser although we specified the
+# myst_heading_anchors option.
+#
+# When linking to sections within the same document, MyST may generate warnings
+# such as:
+#
+#   WARNING: 'myst' cross-reference target not found: 'link' [myst.xref_missing]
+#
+# This warning occurs because MyST parser could not refer to the heading targets
+# when generating the targets in building steps. Despite the warning, the link
+# targets are correctly rendered in the final documentation.
+#
+# If the behavior of MyST or our handling of anchor generation changes in the
+# future, this workaround may need to be revisited.
+suppress_warnings = [
+  "myst.xref_missing"
+]
```

Comment:
Can we specify meaningful simple anchor like we did in https://raw.githubusercontent.com/groonga/groonga/refs/heads/main/doc/source/news/15.md (`(release-15-0-4)=`) instead?

Reply:
fix: [e64c7a1](https://github.com/groonga/groonga/pull/2271/commits/e64c7a1693e134f2ae7a6756a0a5c562af0759ae) Thank you so much. we can use the section reference!